src/utils/telemetry/report-error.mjs

```diff
@@ -0,0 +1,52 @@
+import { dirname, join } from 'path'
+import process, { version as nodejsVersion } from 'process'
+import { fileURLToPath } from 'url'
+
+import execa from '../execa.mjs'
+import getGlobalConfig from '../get-global-config.mjs'
+
+import { isTelemetryDisabled, cliVersion } from './utils.mjs'
+
+const dirPath = dirname(fileURLToPath(import.meta.url))
+
+/**
+ *
+ * @param {import('@bugsnag/js').NotifiableError} error
+ * @param {object} config
+ * @param {import('@bugsnag/js').Event['severity']} config.severity
+ * @returns {Promise<void>}
+ */
+export const reportError = async function (error, config = {}) {
+  const globalConfig = await getGlobalConfig()
+
+  if (isTelemetryDisabled(globalConfig)) {
+    return
+  }
+
+  const user = globalConfig.get(`users.${globalConfig.get('userId')}`)
```

Comment:
We should handle logged out errors

Reply:
`user` is undefined if the user is logged out